pkg/vc/issuer/credentials.go

```diff
@@ -89,3 +89,12 @@ func CreateDIDCommInitCredential(docJSON []byte) *verifiable.Credential {
 
 	return vc
 }
+
+// CreatePresentationSubmission creates presentation submission vp.
+func CreatePresentationSubmission() *verifiable.Presentation {
```

Comment:
just curious, why issuer needs to create presentation submission? holder/wallet creates it.

Reply:
This is creating an VP. I tried to align the function name based on the sample resp described in https://github.com/trustbloc/edge-adapter/pull/123#issue-440924663 (Example Issuer response:).

Reply:
@sudeshrshetty issuer adapter needs to create presentation submissions for the rp adapter

Reply:
The `request-presentation` message is coming from the RP adapter

Reply:
@sudeshrshetty [Updated](https://github.com/trustbloc/edge-adapter/pull/140/files#diff-2328620458fc9d4a14b1f25ec1e0fd38R94) the function name. As discussed with @sudeshrshetty and @llorllale , the request between rp and issuer will not contain presentation definitions.